plugins/modules/aci_l3out_hsrp_interface_profile.py

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Shreyas Srish (@shrsr) <ssrish@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l3out_hsrp_interface_profile
+short_description: Manages Layer 3 Outside (L3Out) HSRP interface profile (hsrpIfP)
+description:
+- Manages L3Out HSRP interface profile on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - Name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+    required: true
+  l3out:
+    description:
+    - Name of an existing L3Out.
+    type: str
+    aliases: [ l3out_name ]
+    required: true
+  node_profile:
+    description:
+    - Name of the node profile.
+    type: str
+    aliases: [ node_profile_name, logical_node ]
+    required: true
+  interface_profile:
+    description:
+    - Name of an existing interface profile.
+    type: str
+    aliases: [ name, interface_profile_name, logical_interface ]
+    required: true
+  hsrp_policy:
+    description:
+    - Name of an existing hsrp interface policy.
+    type: str
+    aliases: [ name, hsrp_policy_name ]
+  version:
+    description:
+    - The version of the compatibility catalog.
+    type: str
+    choices: [ v1, v2 ]
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+extends_documentation_fragment:
+- cisco.aci.aci
+- cisco.aci.annotation
+- cisco.aci.owner
+
+notes:
+- The C(tenant), C(l3out), C(logical_node_profile) and C(logical_interface_profile) must exist before using this module in your playbook.
+  The M(cisco.aci.aci_tenant), M(cisco.aci.aci_l3out), M(cisco.aci.aci_l3out_logical_node_profile) and M(cisco.aci.aci_l3out_logical_interface_profile) \
```

Comment:
missing hsrp_policy

Reply:
It's not a required option for this module to exist